src/browser/starter.js

```diff
@@ -1174,7 +1169,7 @@ V86Starter.prototype.mount_fs = async function(path, baseurl, basefs, callback)
  * @param {function(Object)=} callback
  * @export
  */
-V86Starter.prototype.create_file = function(file, data, callback)
+V86Starter.prototype.create_file = async function(file, data, callback)
```

Comment:
We should probably remove the callback and return a promise instead.

Reply:
@copy  i have modified code